### ISSUE TITLE
Duplicate log messages sent after 2.2.0

### ISSUE THREAD

**Issue by roberthannon** [SumoLogic/js-sumo-logger#65]
Hi, I think an issue was introduced in #64 which can cause duplicate log messages to be sent to sumo. If I revert to version 2.1.1, the issue no longer occurs.

The problem seems to be caused by the addition of `this.startLogSending();` to:

```
            return axios
                .post(this.config.endpoint, this.pendingLogs.join('\n'), {
                    headers
                })
                .then(() => {
                    this.pendingLogs = [];
                    // Reset interval if needed:
                    this.startLogSending();
                    this.config.onSuccess();
                })
                .catch((error) => {
                    this.config.onError(error);
                });
```

I think the problem is with `startLogSending` itself - it calls `setInterval` without first calling `clearInterval` and so ends up with duplicate timers executing...

**Comment by billsaysthis** [comment]
Thanks for the bug report, on it.

**Comment by billsaysthis** [comment]
Please pull the new release and let me know if it fixes your issue.

**Comment by roberthannon** [comment]
Thanks for the quick response! I can confirm the issues seems to be fixed as of 2.3.1
